contracts/Vault.vy

```diff
@@ -956,8 +963,14 @@ def withdraw(_shares: uint256 = MAX_UINT256, recipient: address = msg.sender) ->
 
     if value > self.token.balanceOf(self):
         # We need to go get some from our strategies in the withdrawal queue
-        # NOTE: This performs forced withdrawals from each Strategy. There is
-        #       a 0.5% withdrawal fee assessed on each forced withdrawal (<= 0.5% total)
```

Comment:
is the fee no longer considered? if maxLoss is 0 should it account for possible fees

Reply:
Yeah, there is no fee. That's one of the audit comments. We forgot this little piece